Propose tweak to vertical gutters documentation (#36996)
twbs/bootstrap: site/content/docs/5.2/layout/gutters.md

## Vertical gutters

> `.gy-*` classes can be used to control the vertical gutter widths for a **single** row where overflowing columns are [wrapped onto a new line](https://getbootstrap.com/docs/5.2/layout/columns/#column-wrapping). Like the horizontal gutters, the vertical gutters can cause some overflow below the `.row` at the end of a page. If this occurs, you add a wrapper around `.row` with the `.overflow-hidden` class:

Reviewer: ```suggestion
`.gy-*` classes can be used to control the vertical gutter widths within a row when columns wrap to new lines. Like the horizontal gutters, the vertical gutters can cause some overflow below the `.row` at the end of a page. If this occurs, you add a wrapper around `.row` with the `.overflow-hidden` class:
```
